src/ExpressiveInstaller/config.php

```diff
@@ -167,10 +200,22 @@
                     ],
                     'copy-files' => [
                         '/Resources/config/templates-zend-view.php' => '/config/autoload/templates.global.php',
-                        '/Resources/templates/zend-view-404.phtml' => '/src/App/templates/error/404.phtml',
-                        '/Resources/templates/zend-view-error.phtml' => '/src/App/templates/error/error.phtml',
-                        '/Resources/templates/zend-view-layout.phtml' => '/src/App/templates/layout/default.phtml',
-                        '/Resources/templates/zend-view-home-page.phtml' => '/src/App/templates/app/home-page.phtml',
+                        '/Resources/templates/zend-view-404.phtml' => [
+                            'flat'    => '/templates/error/404.phtml',
+                            'modular' => '/src/App/templates/error/404.phtml',
+                        ],
+                        '/Resources/templates/zend-view-error.phtml' => [
+                            'flat'    => '/templates/error/error.phtml',
+                            'modular' => '/src/App/templates/error/error.phtml',
+                        ],
+                        '/Resources/templates/zend-view-layout.phtml' => [
+                            'flat'    => '/templates/layout/default.phtml',
+                            'modular' => '/src/App/templates/layout/default.phtml',
+                        ],
+                        '/Resources/templates/zend-view-home-page.phtml' => [
+                            'flat'    => '/templates/app/home-page.phtml',
+                            'modular' => '/src/App/templates/app/home-page.phtml',
+                        ],
                     ],
                     'minimal-files' => [
                         '/Resources/config/templates-zend-view.php' => '/config/autoload/templates.global.php',
```

Comment:
I thought of that as I started doing this, but was having trouble figuring out a clean way to make it happen. I'll see what I can come up with today.